src/content/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/install-prometheus-agent.mdx

```diff
@@ -5,4 +5,174 @@ tags:
   - Prometheus integrations
   - Install and configure Prometheus agent
 metaDescription: 'How to install, update, or uninstall your New Relic Prometheus agent alongside the Kubernetes integration.'
----
\ No newline at end of file
+---
+
+import infrastructurePrometheusIoDashboard from 'images/infrastructure_screenshot-crop_prometheus-io-dashboard.png'
+
+import infrastructurePrometheusNrqlQuery from 'images/infrastructure_screenshot-crop_prometheus-nrql-query.png'
+
+New Relic's Prometheus agent is a thin wrapper on top of Prometheus server to run it in agent mode. With this integration, you can create a YAML configuration file used by Prometheus that's ready to send metrics to New Relic.
+
+Learn more about the solution from [its repository](https://github.com/newrelic/newrelic-prometheus-configurator).
+
+## Install the Prometheus agent alongside the Kubernetes integration [#kubernetes-install]
+
+You can install our [Kubernetes integration](/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration) to get a full observability of your Kubernetes clusters, which includes the Prometheus agent.
+
+1. Install our Kubernetes integration. See how to install it [here](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure).
```

Comment:
We could mention that prometheus-agent should be enabled when installing the Kubernetes integration.